app/models/email.rb

```diff
@@ -56,7 +56,7 @@ def self.email_available?(email, community_id)
   end
 
   def self.send_confirmation(email, community)
-    MailCarrier.deliver_later(PersonMailer.email_confirmation(email, community))
+    Delayed::Job.enqueue(EmailConfirmationJob.new(email.id, community.id), priority: 2)
```

Comment:
I guess this commit message explains the thinking: https://github.com/sharetribe/sharetribe/commit/b0a36a006c10e19900c4adb91f2f896723a67dda .